vendor/hound/config/style_guides/ruby.yml

```diff
@@ -102,7 +101,7 @@ Style/StringLiterals:
   Description: Checks if uses of quotes match the configured preference.
   StyleGuide: https://github.com/bbatsov/ruby-style-guide#consistent-string-literals
   Enabled: true
-  EnforcedStyle: double_quotes
```

Comment:
I think we should keep the `double_quotes` here, there's no need to change quotes style in the entire codebase.

Reply:
Fine. I'll change that back.
What is your opinion on DotPosition?

Reply:
> What is your opinion on DotPosition?

I like it. 👍